src/debug.rs

```diff
@@ -33,7 +35,7 @@ pub trait DebugQueryTable {
 pub struct TableEntry<K, V> {
     /// key of the query
     pub key: K,
-    /// value of the query, if it is stored
+    /// value of the query, if it =is stored
```

Comment:
(I have a few trivial nits. Sorry if this PR isn’t ready for review yet)

Typo - errant “=“

Reply:
```suggestion
    /// value of the query, if it is stored
```